build.sbt

```diff
@@ -1,5 +1,6 @@
 lazy val root = (project in file(".")).settings(
   sbtPlugin := true,
+  crossSbtVersions := Vector("0.13.16", "1.0.2"),
   name := "sbt-diff-project",
   licenses += "MIT" -> url("https://raw.githubusercontent.com/opt-tech/sbt-diff-project/master/LICENSE"),
   addSbtPlugin("com.mayreh" % "sbt-reverse-dependency" % "0.1.1"),
```

Comment:
I need to update this plugin once https://github.com/ocadaruma/sbt-reverse-dependency/pull/1 is merged and a new version is released.